pkg/cli/common.go

```diff
@@ -239,6 +240,7 @@ func GetBudFlags(flags *BudResults) pflag.FlagSet {
 		panic(fmt.Sprintf("error marking the pull-never flag as hidden: %v", err))
 	}
 	fs.BoolVarP(&flags.Quiet, "quiet", "q", false, "refrain from announcing build instructions and image read/write progress")
+	fs.BoolVar(&flags.AddHistory, "add-history", true, "add history for build layers (default true)")
```

Comment:
(default true) here I believe will cause a stutter. if you do buildah build --help.  Please remove it as well as the line below 
--identity already stutters.

```
buildah build --help
...
      --identity-label                add default identity label (default true) (default true)
```